it/uk/gov/hmrc/agentmappingfrontend/support/EndpointBehaviours.scala

```diff
@@ -28,80 +28,62 @@ trait EndpointBehaviours extends AuthStubs {
       auditEventShouldNotHaveBeenSent("CheckAgentRefCode")
     }
 
-    "redirect to the start page if the current user is logged in and does not have affinity group Agent" in {
-      givenUserIsAuthenticated(individual)
+    "redirect to the sign-in page if the current user is logged in but does not have affinity group Agent" in {
+      givenUnauthorisedWith("UnsupportedAffinityGroup")
       val request = fakeRequest(endpointMethod,endpointPath)
       val result = await(doRequest(request))
 
       result.header.status shouldBe 303
```

Comment:
redirectLocation to replace line 36 and 37

Reply:
case ResponseHeader(SEE_OTHER, headers, _) => headers.get(LOCATION)

will do these 2 checks, in 1 line